Description: fix translations
 fix translations - normally upstream does this before creating a
 tar ball
Author: ChangZhuo Chen (陳昌倬) <czchen@debian.org>

---
Last-Update: 2015-12-26

--- /dev/null
+++ lxqt-about-0.10.0/translations/lxqt-about_el_GR.ts
@@ -0,0 +1,178 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!DOCTYPE TS>
+<TS version="2.1" language="el_GR">
+<context>
+    <name>AboutDialogPrivate</name>
+    <message>
+        <location filename="../aboutdialog/lxqtaboutdialog.cpp" line="83"/>
+        <source>Version: %1</source>
+        <translation>Έκδοση: %1</translation>
+    </message>
+    <message>
+        <location filename="../aboutdialog/lxqtaboutdialog.cpp" line="96"/>
+        <source>Advanced, easy-to-use, and fast desktop environment based on Qt technologies.</source>
+        <comment>About dialog, &apos;About&apos; tab text</comment>
+        <translation>Προηγμένη, εύκολη στη χρήση και γρήγορη επιφάνεια εργασίας με βάση τις τεχνολογίες Qt.</translation>
+    </message>
+    <message>
+        <location filename="../aboutdialog/lxqtaboutdialog.cpp" line="98"/>
+        <source>LXQt would not have been possible without the &lt;a %1&gt;Razor-qt&lt;/a&gt; project and its many contributors.</source>
+        <comment>About dialog, &apos;About&apos; tab text</comment>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../aboutdialog/lxqtaboutdialog.cpp" line="100"/>
+        <source>Copyright: %1-%2 %3</source>
+        <comment>About dialog, &apos;About&apos; tab text</comment>
+        <translation>Πνευματικά δικαιώματα: %1-%2 %3</translation>
+    </message>
+    <message>
+        <location filename="../aboutdialog/lxqtaboutdialog.cpp" line="102"/>
+        <source>Homepage: %1</source>
+        <comment>About dialog, &apos;About&apos; tab text</comment>
+        <translation>Αρχική σελίδα: %1</translation>
+    </message>
+    <message>
+        <location filename="../aboutdialog/lxqtaboutdialog.cpp" line="104"/>
+        <source>License: %1</source>
+        <comment>About dialog, &apos;About&apos; tab text</comment>
+        <translation>Άδεια χρήσης: %1</translation>
+    </message>
+    <message>
+        <location filename="../aboutdialog/lxqtaboutdialog.cpp" line="114"/>
+        <source>LXQt is developed by the &lt;a %1&gt;LXQt Team and contributors&lt;/a&gt;.</source>
+        <comment>About dialog, &apos;Authors&apos; tab text</comment>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../aboutdialog/lxqtaboutdialog.cpp" line="116"/>
+        <source>If you are interested in working with our development team, &lt;a %1&gt;join us&lt;/a&gt;.</source>
+        <comment>About dialog, &apos;Authors&apos; tab text</comment>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../aboutdialog/lxqtaboutdialog.cpp" line="134"/>
+        <source>Special thanks to:</source>
+        <comment>About dialog, &apos;Thanks&apos; tab text</comment>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../aboutdialog/lxqtaboutdialog.cpp" line="141"/>
+        <source>LXQt is translated into many languages thanks to the work of the translation teams all over the world.</source>
+        <comment>About dialog, &apos;Translations&apos; tab text</comment>
+        <translation>LXQt έχει μεταφραστεί σε πολλές γλώσσες χάρη στο έργο των μεταφραστικών ομάδων σε όλο τον κόσμο.</translation>
+    </message>
+    <message>
+        <source>Yes</source>
+        <translation type="obsolete">Ναι</translation>
+    </message>
+    <message>
+        <source>No</source>
+        <translation type="obsolete">Όχι</translation>
+    </message>
+    <message>
+        <source>&lt;b&gt;LxQt Desktop Toolbox - Technical Info&lt;/b&gt;</source>
+        <translation type="obsolete">&lt;b&gt;Επιφάνεια εργασίας LXQt - Τεχνικές πληροφορίες&lt;/b&gt;</translation>
+    </message>
+    <message>
+        <source>Version</source>
+        <translation type="obsolete">Έκδοση</translation>
+    </message>
+    <message>
+        <source>Qt</source>
+        <translation type="obsolete">Qt</translation>
+    </message>
+    <message>
+        <source>Debug Build</source>
+        <translation type="obsolete">Κατασκευή αποσφαλμάτωσης</translation>
+    </message>
+    <message>
+        <source>System Configuration</source>
+        <translation type="obsolete">Διαμόρφωση συστήματος</translation>
+    </message>
+    <message>
+        <source>Share Directory</source>
+        <translation type="obsolete">Κοινόχρηστος κατάλογος</translation>
+    </message>
+    <message>
+        <source>Translations</source>
+        <comment>About dialog, Tab title</comment>
+        <translation type="obsolete">Μεταφράσεις</translation>
+    </message>
+    <message>
+        <source>&lt;b&gt;User Directories&lt;/b&gt;</source>
+        <translation type="obsolete">&lt;b&gt;Κατάλογοι χρηστών&lt;/b&gt;</translation>
+    </message>
+    <message>
+        <source>Xdg Data Home</source>
+        <translation type="obsolete">Αρχικός κατάλογος δεδομένων xdg</translation>
+    </message>
+    <message>
+        <source>Xdg Config Home</source>
+        <translation type="obsolete">Αρχικός κατάλογος διαμόρφωσης xdg</translation>
+    </message>
+    <message>
+        <source>Xdg Data Dirs</source>
+        <translation type="obsolete">Κατάλογοι δεδομένων xdg</translation>
+    </message>
+    <message>
+        <source>Xdg Cache Home</source>
+        <translation type="obsolete">Αρχικός κατάλογος λανθάνουσας μνήμης xdg</translation>
+    </message>
+    <message>
+        <source>Xdg Runtime Home</source>
+        <translation type="obsolete">Αρχικός κατάλογος εκτέλεσης xdg</translation>
+    </message>
+    <message>
+        <source>Xdg Autostart Dirs</source>
+        <translation type="obsolete">Κατάλογοι αυτόματης εκκίνησης xdg</translation>
+    </message>
+    <message>
+        <source>Xdg Autostart Home</source>
+        <translation type="obsolete">Αρχικός κατάλογος αυτόματης εκκίνησης xdg</translation>
+    </message>
+</context>
+<context>
+    <name>about</name>
+    <message>
+        <location filename="../aboutdialog/lxqtaboutdialog.ui" line="14"/>
+        <source> About LXQt</source>
+        <translation>Σχετικά με το LXQt</translation>
+    </message>
+    <message>
+        <location filename="../aboutdialog/lxqtaboutdialog.ui" line="66"/>
+        <source>About</source>
+        <comment>About dialog, Tab title </comment>
+        <translation>Σχετικά</translation>
+    </message>
+    <message>
+        <location filename="../aboutdialog/lxqtaboutdialog.ui" line="83"/>
+        <source>Authors</source>
+        <comment>About dialog, Tab title</comment>
+        <translation>Δημιουργοί</translation>
+    </message>
+    <message>
+        <location filename="../aboutdialog/lxqtaboutdialog.ui" line="100"/>
+        <source>Thanks</source>
+        <comment>About dialog, Tab title</comment>
+        <translation>Ευχαριστίες</translation>
+    </message>
+    <message>
+        <location filename="../aboutdialog/lxqtaboutdialog.ui" line="117"/>
+        <source>Translations</source>
+        <comment>About dialog, Tab title</comment>
+        <translation>Μεταφράσεις</translation>
+    </message>
+    <message>
+        <location filename="../aboutdialog/lxqtaboutdialog.ui" line="134"/>
+        <source>Technical Info</source>
+        <comment>About dialog, Tab title</comment>
+        <translation>Τεχνικές πληροφορίες</translation>
+    </message>
+    <message>
+        <location filename="../aboutdialog/lxqtaboutdialog.ui" line="165"/>
+        <source>Copy to clipboard</source>
+        <translation type="unfinished"></translation>
+    </message>
+</context>
+</TS>
--- /dev/null
+++ lxqt-about-0.10.0/translatorsinfo/translators_el_GR.info
@@ -0,0 +1,42 @@
+# _help = Don't translate this text, it is only help. I want to thank you in the "About" dialog. So, please fill the following information about yourself. The number does not matter, all the names will be displayed in alphabetical order.
+
+
+translator_1_nameEnglish = Efstathios Iosifidis
+translator_1_nameNative  = Ευστάθιος Ιωσηφίδης
+translator_1_contact     = iosifidis@opensuse.org
+
+translator_2_nameEnglish = Antonis Geralis
+translator_2_nameNative  = Αντώνης Γέραλης
+translator_2_contact     = capoiosct@gmail.com
+
+# translator_3_nameEnglish = Translator 3. Your name in English. 
+# translator_3_nameNative  = Translator 3. Your name in the native language.
+# translator_3_contact     = Translator 3. Contact information, email or web site address.
+
+# translator_4_nameEnglish = Translator 4. Your name in English. 
+# translator_4_nameNative  = Translator 4. Your name in the native language.
+# translator_4_contact     = Translator 4. Contact information, email or web site address.
+
+# translator_5_nameEnglish = Translator 5. Your name in English. 
+# translator_5_nameNative  = Translator 5. Your name in the native language.
+# translator_5_contact     = Translator 5. Contact information, email or web site address.
+
+# translator_6_nameEnglish = Translator 6. Your name in English. 
+# translator_6_nameNative  = Translator 6. Your name in the native language.
+# translator_6_contact     = Translator 6. Contact information, email or web site address.
+
+# translator_7_nameEnglish = Translator 7. Your name in English. 
+# translator_7_nameNative  = Translator 7. Your name in the native language.
+# translator_7_contact     = Translator 7. Contact information, email or web site address.
+
+# translator_8_nameEnglish = Translator 8. Your name in English. 
+# translator_8_nameNative  = Translator 8. Your name in the native language.
+# translator_8_contact     = Translator 8. Contact information, email or web site address.
+
+# translator_9_nameEnglish = Translator 9. Your name in English. 
+# translator_9_nameNative  = Translator 9. Your name in the native language.
+# translator_9_contact     = Translator 9. Contact information, email or web site address.
+
+# translator_10_nameEnglish = Translator 10. Your name in English. 
+# translator_10_nameNative  = Translator 10. Your name in the native language.
+# translator_10_contact     = Translator 10. Contact information, email or web site address.
